packages/in_app_purchase/pubspec.yaml

```diff
@@ -1,30 +1,26 @@
 name: in_app_purchase
 description: A Flutter plugin for in-app purchases. Exposes APIs for making in-app purchases through the App Store and Google Play.
 homepage: https://github.com/flutter/plugins/tree/master/packages/in_app_purchase
-version: 0.3.5+2
+version: 0.4.0-nullsafety.0
 
 dependencies:
-  async: ^2.0.8
-  collection: ^1.14.11
   flutter:
     sdk: flutter
-  json_annotation: ^3.0.0
-  meta: ^1.1.6
+  json_annotation: ^4.0.0-nullsafety.0
```

Comment:
Just 4.0.0 now. In fact, it looks like all of the `-nullsafe` versions here now have stable previews you can use.

Reply:
done